app/lib/actions/katello/agent_action.rb

```diff
@@ -28,6 +28,8 @@ def plan(host, options)
       end
 
       def run(event = nil)
+        require 'pry'
```

Comment:
Looks like something went wrong in this last push

Reply:
Outdated, resolved.